### ISSUE TITLE
[2022.2.9.19.55.57] Numpy compatibility with Linux wheel

### ISSUE THREAD

**Issue by rphel** [moble/quaternion#194]
**Describe the bug**
`numpy-quaternion` is not compatible of `numpy` prior to v1.22.0

**To Reproduce**

```console
$ pip install 'numpy~=1.21.0'
$ pip install 'numpy-quaternion==2022.2.9.19.55.57'
$ python -c 'import quaternion'
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/test4/lib/python3.8/site-packages/quaternion/__init__.py", line 27, in <module>
    from .numpy_quaternion import (
ImportError: numpy.core.multiarray failed to import
$ pip install -U 'numpy~=1.22.0'
$ python -c 'import quaternion' 
/tmp/test4/lib/python3.8/site-packages/quaternion/numba_wrapper.py:23: UserWarning: 

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Could not import from numba, which means that some
parts of this code may run MUCH more slowly.  You
may wish to install numba.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

  warnings.warn(warning_text)
/tmp/test4/lib/python3.8/site-packages/quaternion/calculus.py:477: UserWarning: 

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Could not import from scipy, which means that derivatives
and integrals will use less accurate finite-differencing
techniques.  You may want to install scipy.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

  warnings.warn(warning_text)
```

**Expected behavior**
Numpy version greater than 1.13.0 should work.

**Environment (please complete the following information):**
 - OS/Python: Ubuntu, Python 3.8.10
 - Tools: pip, wheels
 - Numpy version: <=1.21.x
 - numpy-quaternion: 2022.2.9.19.55.57

**Comment by moble** [comment]
It now works.  Each wheel is built with the oldest supported version of numpy for that python version and architecture, and then tested against the newest version available.  Thanks again!

Closed by c22f93840a7ab7391576085b92e2545583f9cb40.

**Comment by rphel** [comment]
Hello Mike. Thank you very much for this quick fix. I confirm it works now like a charm.
